src/components/10-atoms/input-file/index.js

```diff
@@ -51,6 +53,7 @@ class AXAInputFile extends NoShadowDOM {
 
   render() {
     const {
+      text,
```

Comment:
In some cases (which i do not know right now) we need a default value for strings here. So please write `text = ''`